MCLauncher/MainWindow.xaml

```diff
@@ -59,8 +59,8 @@
             </Grid>
         </DataTemplate>
     </Window.Resources>
-    <Grid>
-        <ListBox Name="VersionList" Margin="10,10,10,10"  d:DataContext="{d:DesignData /SampleData/Versions.xaml}" ItemsSource="{Binding}" HorizontalContentAlignment="Stretch">
+    <Grid Margin="0,0,0,29">
```

Comment:
Would be nice if this Margin could be removed from here and replaced with nice numbers in the ListBox instead.

Reply:
I kept a 10,10,10,10 on the grid for consistency's sake. The rest was to make space for the bottom grid row, which I've cleaned up.